main.go

```diff
@@ -13,14 +12,15 @@ import (
 
 	"github.com/BurntSushi/toml"
 	options "github.com/mreiferson/go-options"
+	"github.com/pusher/oauth2_proxy/logger"
 )
 
 func main() {
-	log.SetFlags(log.Ldate | log.Ltime | log.Lshortfile)
+	logger.SetFlags(logger.Lshortfile)
 	flagSet := flag.NewFlagSet("oauth2_proxy", flag.ExitOnError)
 
 	emailDomains := StringArray{}
-	whitelistDomains := StringArray{}
+	whitelistandardomains := StringArray{}
```

Comment:
Any idea why this variable has been renamed?

Reply:
Merge mistake - I was looking for instances of "std" for some reason I can't immediately recall and I rolled over this. I've fixed it now.